indico/modules/events/contributions/models/contributions.py

```diff
@@ -308,10 +309,14 @@ def protection_parent(self):
     def track(self):
         return self.event_new.as_legacy.getTrackById(str(self.track_id))
 
-    @property
+    @hybrid_property
     def start_dt(self):
         return self.timetable_entry.start_dt if self.timetable_entry else None
 
+    @start_dt.expression
+    def start_dt(self):
+        return self.timetable_entry.start_dt
```

Comment:
Ah, I don't need it anymore. Apparently I didn't remove it after I found out about the `is_scheduled` property.